frontend/insurance/src/app/shared/interfaces/VinApi.ts

```diff
@@ -0,0 +1,9 @@
+export interface VinApi {
+  Results: Array<Results>;
+}
+
+interface Results {
+  Model: string;
+  ModelYear: string;
+  BasePrice: string;
+}
```

Comment:
Did you delete the interface I had made previously?

Reply:
No, this is for the response from the vehicle API. It is mainly so there aren't any dumb typescript strict errors.